genestack_client/scripts/genestack_application_manager.py

```diff
@@ -103,8 +103,8 @@ def update_parser(self, p):
                  '`-i <group_accession>` for group visibility)'
         )
         p.add_argument(
-            '-w', '--wait', action='store_true',
-            help='wait until all installed applications will be completely loaded'
+            '-n', '--no-wait', action='store_true', dest='no_wait',
+            help='Don\'t wait until all installed applications will be completely loaded'
```

Comment:
We use `cannot` in such cases.

PS. Double quotes should be used to avoid escaping.  https://trac.genestack.com/wiki/DevStuff/CodeStyle/Python